specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json

```diff
@@ -622,24 +682,21 @@
       "type": "string",
       "description": "Required. Indicates the type of cognitive service account.",
       "enum": [
-        "Academic",
-        "Bing.Autosuggest",
```

Comment:
Academic - decommissioned from Ibiza to Labs.
Bing.Autosuggest/Search/SpellCheck - replaced by their .v7 version.
Recommendations - decommissioned.
Speech - decommissioned, replaced by Bing.Speech.